napari_animation/_tests/fixtures.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from napari_animation import Animation
+
+@pytest.fixture
+def make_napari_viewer():
```

Comment:
curious why you're copying this here?  you already have napari in your test dependencies, and it will provide this

Reply:
ah, maybe this was before 0.4.5 was released, where I think things were a little in flux (or maybe i just didn't know where things were) we should update this now though, agreed

Reply:
I'll make sure to import it - thanks!